tests/installation/first_boot.pm

```diff
@@ -40,7 +40,7 @@ sub run() {
         }
         else {
             send_key "ret";
```

Comment:
I was watching the installation and this "ret" was pressed before user was visible, so no password prompt at all